uat/testing-features/src/main/java/com/aws/greengrass/LocalDeploymentSteps.java

```diff
@@ -119,6 +132,23 @@ private CommandInput prepareCliDeploymentCommand(String componentName, String co
                 .build();
     }
 
+    private CommandInput getCliUpdateConfigurationCommand(String componentName,
+                                                     List<Map<String, String>> configuration) throws IOException {
+        List<String> commandArgs = Arrays.asList(
+                "deployment",
+                "create",
+                "--merge " + componentName));
```

Comment:
merge arg takes both component name-version. I don't see version here. 
But if you only want to update the configuration of the locally installed components, you don't need `--merge` arg (which is what you're doing here). 

You can reuse the `prepareCliDeploymentCommand` and check for version. If it is null, do not include `--merge` arg.

Reply:
done.